appinventor/appengine/src/com/google/appinventor/client/editor/simple/palette/SimpleComponentDescriptor.java

```diff
@@ -519,6 +531,8 @@ public static MockComponent createMockComponent(String name, String type, Simple
       return new MockFeatureCollection(editor);
     } else if (name.equals(MockChart.TYPE)) {
       return new MockChart(editor);
+    } else if (name.equals(MockImage.TYPE)) {
+      return new MockChart(editor);
```

Comment:
????

Reply:
Yeah ??? Not sure where this came from, but I flushed it. It turns out it is harmless because the check for MockImage.TYPE already happened and if true the function returned. But, yeah. It's gone now.